src/js/sjcl-custom/sjcl-ecdsa-signdeterministic.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+/*global sjcl*/
+/*eslint-disable new-cap*/
+
+function generateK(secretKey, hash, hashObject) {
```

Comment:
This function doesn't appear to match the algorithm described in http://tools.ietf.org/html/rfc6979#section-3.2